rsconcept/backend/shared/messages.py

```
@@ -2,209 +2,217 @@
+def parentNotInOSS() -> str:
     return 'Родительский блок не принадлежит ОСС'
 
 
-def blockCyclicHierarchy():
+def blockCyclicHierarchy() -> str:
     return 'Попытка создания циклического вложения'
 
 
-def childNotInOSS():
+def childNotInOSS() -> str:
     return f'Дочерний элемент блок не принадлежит ОСС'
```

Ruff (0.15.15)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)